index.html

```diff
@@ -30,5 +30,136 @@ <h2>I’am a Software Developer</h2>
             <li><a href=""><span class="m-icon"></span></a></li>
         </ul>
     </section>
+    <section class="works">
+        <h1>My Recent Works</h1>
+        <section class="featured-work">
+            <img class="featured-img" src="img/img-placeholder.png" alt="featured-project">
+            <h2>Multi-Post Stories</h2>
+            <p>
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text
+                 ever since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+            <ul class="tags">
+                <li><a class="tags-elements" href="">css</a></li>
+                <li><a class="tags-elements" href="">html</a></li>
+                <li><a class="tags-elements" href="">bootstrap</a></li>
+                <li><a class="tags-elements" href="">Ruby</a></li>
+            </ul>
```

Comment:
- [ ] I'm so impressed with how you used the correct tags `ul` and `li` here 🔥. However, please remove the `a` tag from the lists as the list here is just a list of technologies and not a link 😄.